### DISCUSSION TITLE
Aliquote IVA in inglese

### DISCUSSION THREAD

**Discussion by luca-libonati** [fattureincloud/api#352] [Q&A]
Buonsera,
c'è un modo per avere la versione in inglese della frase che viene aggiunta in fattura relativa all'aliquota iva selezionata?
In alternativa, c'è un modo per generare questa frase disabilitando quella di default?
Grazie mille,
L.

**Comment by emajo** [comment]
Ciao @luca-libonati, le aliquote già presenti negli account Fatture in Cloud hanno tutte le descrizioni in italiano, per averle in inglese è neccessario creare delle aliquote apposite attraverso la chiamata [Create Vat Type](https://developers.fattureincloud.it/api-reference#post-/c/-company_id-/settings/vat_types) e passare i relativi id in creazione di documento.

Per ricevere supporto su funzionalità più specifiche di prodotto le consigliamo di contattare il nostro supporto clienti come specificato nella [nostra pagina relativa al supporto](https://developers.fattureincloud.it/docs/support).

Grazie,
Emanuele.

**Reply by valmoz** [reply]
Buongiorno @luca-libonati abbiamo risposto alla sua domanda?

Grazie
Mauro
